utilities/harness-codefresh

```diff
@@ -15,14 +15,18 @@ if __name__ == "__main__":
 
     parser = argparse.ArgumentParser(
         description='Walk filesystem inside ./applications and define build and deploy scripts.')
-    parser.add_argument('path', metavar='path', default=ROOT, type=str,
-                        help='Base path of the application.')
+    parser.add_argument('paths', metavar='paths', default=[ROOT], type=str, nargs='*',
+                        help='Base paths to start looking for applications. The paths will be processed '
+                             'in the given order so determining the build and overriding precedence.')
+    parser.add_argument('-i', '--include', dest='include', action="append", default=[],
```

Comment:
very nice to have -i here too!

except one thing: do we need to add the base images to the -i too or will all base images always be build?
I btw. expect all base images always to be build ;-)

Reply:
this is at the moment behaving exactly as the harness-deployment counterpart, so if no -i is specified everything is built. Agree base images being included by default would be an improvement but I'd prefer to solve that with the application dependency definition rather than doing something now

Reply:
okay